config/application.rb

```diff
@@ -22,8 +22,14 @@ class Application < Rails::Application
 
     # Do not swallow errors in after_commit/after_rollback callbacks.
     config.active_record.raise_in_transactional_callbacks = true
-    
+
     # Gzip files when possible
     config.middleware.use Rack::Deflater
+
+    #HCaseflow application urls
+    config.app_urls = {"Caseflow" => "https://caseflow.ds.va.gov", "Efolder" => "https://efolder.cf.ds.va.gov"}
```

Comment:
it's not hard to be a little liberal with what we accept here— and most urls we construct are entirely lowercase— so at the very least, we shouldn't require the app name to be capitalized